### DISCUSSION TITLE
queryString and hash

### DISCUSSION THREAD

**Discussion by nikgodda** [livewire/livewire#4907] [1. Help]
I use Livewire query string [https://laravel-livewire.com/docs/2.x/query-string](https://laravel-livewire.com/docs/2.x/query-string) 
When i add hash to url using pushState Livewire removes it. How to avoid it?

**Comment by joshhanley** [comment]
@ttfbjij Livewire controls the query string, so you can't use pushState otherwise Livewire overwrites it. Instead you will need to add the hash to the query string by using Livewire to do it.

Hope this helps!
